src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -901,8 +901,9 @@ private void testResponseRate() {
 
         // Change the feedback path of the question and save
         feedbackEditPage = getFeedbackEditPage();
-        // Alert class should be enabled to warn instructors against changing visibility options after collecting responses.
-        // Passing 1 as question number as only one question is created.
+
+        ______TS("check warning against changing visibility options for question with existing response");
```

Comment:
Move this before the `// Change the feedback path...` line. Also, make this edit:
... ~~for~~ **while editing** question...